Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1196,6 +1389,30 @@ private void AddLinks()
             }
         }
 
+        private void HandleIgnoreCollider()
```

Comment:
Can you please explain why you are doing this?

Reply:
This part is needed when BB is used with an object that has a rigid body. Because of the BB handle's collider, the object is slightly lifted up from the floor(or any colliderable plane) by the 1/2 amount of the collider size. This is an optional field that developers can specify the plane to ignore collision. (might be important for the situation when the object needs to be placed perfectly on the ground + uses BB handles.

Reply:
yes